bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -1151,8 +1151,8 @@
     </changeSet>
     <changeSet id="bahmni-core-201906101318" author="Vinay, Indraneel, Samuel34">
         <preConditions onFail="MARK_RAN">
-        	<sqlCheck expectedResult="0">
-        		select count(*) from global_property where property = 'order.durationUnitsConceptUuid';
+        	<sqlCheck expectedResult="1">
+        		select count(*) from global_property where property = 'order.durationUnitsConceptUuid' and property_value IS NULL;
```

Comment:
Why is the additional clause needed? if the property existed without a value, this will try to insert the same "property" again, which will fail. If you must run this changeset again in an idempotent manner, then you should check if the property exists, and if it does, then update instead of insert